environment.yml

```diff
@@ -19,6 +19,7 @@ dependencies:
   - jupytext==1.14.4
   - pip>=23.0
   - aesara=2.8.12
+  - pyhf_pymc=0.0.1
```

Comment:
As `pyhf-pymc` isn't on conda-forge this will break your environment file as the solve will fail, so this should be removed.